aquadoggo/src/db/models/query.rs

```diff
@@ -45,6 +45,10 @@ pub struct QueryRow {
     #[sqlx(default)]
     pub value: Option<String>,
 
+    /// Data field used to store blob_piece_v1 bytes
+    #[sqlx(default)]
+    pub data: Option<Vec<u8>>,
```

Comment:
Interesting pattern! I wonder if we have other types there one day as well.

I'd name this column `value_bytes` to underline it's origin from an operation field and maybe keep the door open to have other field types as well.